tests/e2e/salary-template.spec.js

```
@@ -405,14 +577,86 @@ test.describe.serial( 'PR #48: 給与テンプレートと一括登録（仕様
+					} catch ( err ) {
+						// term create に失敗した場合はエラー集約に追加し、他 term の復元は継続。
+						restorationErrors.push(
+							`term restore failed for "${ t.slug || t.name }": ${ err.message }`
+						);
+					}
 				}
 			}
+
+			// 復元中に発生したエラーがあれば、まとめて throw して可視化する。
+			if ( restorationErrors.length > 0 ) {
+				throw new Error( restorationErrors.join( '\n' ) );
```

Biome (2.4.14)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)